### PR TITLE
Examples: Merge animation "keyframes" and "scene" examples.

### PR DESCRIPTION
**Pull request by donmccurdy**
Following up from #14629 and https://github.com/mrdoob/three.js/issues/12903#issuecomment-405050550 —

| ~keyframes/json~ | ~scene~ | keyframes |
|---|---|---|
| screen shot 2018-08-04 at 11 09 40 pm | screen shot 2018-08-04 at 11 09 42 pm | screen shot 2018-08-04 at 11 18 14 pm |

The final model, `LittlestTokyo.glb`, weighs 4 MB:

- 55 MB original
- 19 MB after removing duplicate textures
- 8.5 MB after Draco compression
- 4 MB after optimizing textures

The last two steps are lossy, but visually it looks very close to the original. I'll remove dat.gui if the colors are OK.

Demo: https://rawgit.com/donmccurdy/three.js/examples-keyframes-glb-v2/examples/?q=animation#webgl_animation_keyframes

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

**Comment by takahirox** [comment]
> I'll remove dat.gui if the colors are OK.

I think removing is OK.

**Comment by takahirox** [comment]
This model may be a bit too heavy to render...? 25 fps on my Windows + Chrome. Fine on yours?

File: examples/webgl_animation_keyframes.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+			};
+
+
+			function animate() {
+
+				requestAnimationFrame( animate );
+
+				var delta = clock.getDelta();
+				mixer.update( delta );
+				controls.update( delta );
```
Comment (takahirox) [inline examples/webgl_animation_keyframes.html]: `controls.update()` doesn't take any argument.

https://threejs.org/docs/#examples/controls/OrbitControls.update

And in this example, you might not need to call `.update()` in each frame. by any chance..?

**Comment by looeee** [comment]
Textures are rendered as black on my moto G3, even after giving it a few minutes to load. Also getting around 6fps but the phone is very old and gets a similar fps on plenty of other examples so I don't think that's really an issue. 

What format are the textures in? 

screenshot

**Comment by donmccurdy** [comment]
Thanks for testing — the textures are in PNG and JPG format. Most are 1024x1024, but the main baseColor texture is 4096x4096 JPG, and it looks like that's the one missing on your phone.

Tested this on three devices so far:

- 2012 MBP, 60fps
- iPhone SE, 60fps
- Nexus 5X, 25-30fps

I'll see if I can improve the performance on the Nexus 5X, and maybe that helps @takahirox's machine too. This is probably too much for a moto g3, we'd much simpler model, but it would be nice to get the texture working.

**Comment by mrdoob** [comment]
Yup! The new example looks great and the colors are okay so dat.gui can be removed 👌

**Comment by mrdoob** [comment]
While we're at it, I guess we can delete these?

```
models/json/pump/pump.json
models/json/scene-animation.json
```

**Comment by donmccurdy** [comment]
✅

**Comment by mrdoob** [comment]
Thanks!

**Comment by donmccurdy** [comment]
@looeee does this version of the model display the texture on your phone? https://gltf-viewer.donmccurdy.com/#model=https://storage.googleapis.com/donmccurdy-static/LittlestTokyo2K.glb

**Comment by donmccurdy** [comment]
And here's a version using unlit materials: https://gltf-viewer.donmccurdy.com/#model=https://storage.googleapis.com/donmccurdy-static/LittlestTokyoUnlit.glb

The 2048x2048 texture doesn't improve framerate for me, but the unlit version improves framerate to 40-50fps on my Nexus 5X (from 25-30fps before).

**Comment by looeee** [comment]
Yes, that one loads correctly. I actually prefer the look of the unlit scene, it's more cartoony which suits the style of the model.

**Pull request merged**